src/common/tests/btree.c

```diff
@@ -997,13 +899,100 @@ static struct option btr_ops[] = {
 	{ NULL,		0,			NULL,	0	},
 };
 
+static void
+ts_group(void ** state){
+	int	opt = 0;
+	void	**st = NULL;
+	while ((opt = getopt_long(test_group_stop-test_group_start+1,
```

Comment:
(style)  Missing a blank line after declarations

Reply:
our coding style puts a blank line between declarations at top of function and everything else